packages/tuono/src/config/types.ts

```diff
@@ -10,6 +15,7 @@ export interface TuonoConfig {
   }
   vite?: {
     alias?: AliasOptions
+    css?: CSSOptions
```

Comment:
Please add a reference the the new config option on https://tuono.dev/documentation/configuration

Like other section of the page, this one is a reference to a [`vite` option](https://vite.dev/config/shared-options.html#css-modules), 
so I would like to add it directly in this PR.